src/App.js

```diff
@@ -1,24 +1,31 @@
-import logo from './logo.svg';
+import {
+  BrowserRouter, Routes, Route, Outlet,
+} from 'react-router-dom';
 import './App.css';
+import Navbar from './compoents/Navbar';
+import Book from './compoents/Book';
+import Categories from './compoents/Categories';
+
+function Layout() {
+  return (
+    <>
+      <Navbar />
+      <Outlet />
+    </>
+  );
+}
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <BrowserRouter>
+      <Routes>
+        <Route path="/" element={<Layout />}>
+          <Route index element={<Book title="book1" author="Barry" />} />
+          <Route path="categories" element={<Categories />} />
+          <Route path="*" element={<h1>Not found</h1>} />
+        </Route>
+      </Routes>
+    </BrowserRouter>
   );
 }
```

Comment:
- I advise using arrow functions in order to maintain ES6 syntax as it is the more accepted syntax in use globally. Please fix this 👍 👍